src/test/java/org/folio/list/service/ListServiceValidateReadTest.java

```diff
@@ -37,22 +33,6 @@ void shouldValidateRead() {
     assertDoesNotThrow(() -> validationService.validateRead(entity));
   }
 
-  @Test
-  void validateReadShouldThrowErrorForUserMissingPermissions() {
-    ListEntity entity = TestDataFixture.getListEntityWithoutQuery();
-    when(entityTypeClient.getEntityType(entity.getEntityTypeId()))
-      .thenThrow(new FeignException.Unauthorized("[{\"User is missing permissions: [foo.bar]\"}]", mock(feign.Request.class), null, null));
-    assertThrows(InsufficientEntityTypePermissionsException.class, () -> validationService.validateRead(entity));
-  }
-
-  @Test
-  void validateReadShouldThrowErrorForMissingEntityType() {
-    ListEntity entity = TestDataFixture.getListEntityWithoutQuery();
-    when(entityTypeClient.getEntityType(entity.getEntityTypeId()))
-      .thenThrow(new FeignException.NotFound("Entity type not found", mock(feign.Request.class), null, null));
-    assertThrows(NotFoundException.class, () -> validationService.validateRead(entity));
-  }
-
```

Comment:
These tests are not gone! The logic is moved to `EntityTypeClient`, and there's a new `EntityTypeClientTest`